### DISCUSSION TITLE
How to rotate camera

### DISCUSSION THREAD

**Discussion by ghost** [excaliburjs/Excalibur#2618] [Q&A]
Hi 👋

So I have this space rocket and I wanted to control the camera from within it. The problem is, the camera does not rotate 😓

```TS
/**  Changes speed and rotation of the ship based on fetched data */
public update(engine: Engine, delta: number) {

    super.update(engine, delta)

    // navigation = { zoom: 1, maxZoom: 10, rotation: 0 }
    this.scene.camera.zoom = this.navigation.zoom // ✅ works
    this.scene.camera.rotation = this.navigation.rotation // ❌ 

    this.showActiveJets()
    this.accelerate()
}
```

I thought it would be as simple as zooming in and out, but it doesn't work. 
From the console logs I get, indeed, it changes the value, but that 
👀 doesn't affect how I see the game in a browser at all

I tried with constant values, I've got rid off all other things that affected the camera, still no go

**Comment by eonarheim** [comment]
Hi @adamAfro!

Thanks for posting and sorry for the delay, I think this is bug! 

I've created an issue for this https://github.com/excaliburjs/Excalibur/issues/2619

**Reply by eonarheim** [reply]
I think I have a fix for this, I'll post a PR soon
